src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -513,7 +513,7 @@
         {{ selectedVersion.versionMetadata.engineVersions }}
       </div>
       <div *ngIf="(entryType$ | async) === EntryType.BioWorkflow">
-        <strong matTooltip="Digital Object Identifier">DOI</strong>:
+        <strong matTooltip="Digital Object Identifier">Version DOI</strong>:
```

Comment:
I would just leave this as DOI, although others may disagree (please opine if you do). You're already in a box named "Workflow Version Information", and we don't repeat "version" in front of other labels.

In my mind, it just introduces confusion.. why does it say "Version DOI" but not "Version Description"?

Reply:
>  (please opine if you do)

I don't mind the more descriptive labels.  There's two different kinds of DOIs, and if we include "Concept" and "Version" in the labels, the user (who probably doesn't have our unique point of view and background knowledge) doesn't need to read "DOI", parse the surrounding context, and then reason to determine which type it is).